include/boost/gil/image_processing/scaling.hpp

```diff
@@ -1 +1,65 @@
-#include <boost/gil/image.hpp>
\ No newline at end of file
+#include <boost/gil/image_view.hpp>
+#include <boost/gil/rgb.hpp>
+#include <boost/gil/pixel.hpp>
+#include <boost/gil/image_processing/numeric.hpp>
+
+namespace boost{
+namespace gil{
+template <typename ImageView>
+void lanczos_at(long int source_x, long int source_y, 
+                long int target_x, long int target_y, 
+                ImageView input_view, ImageView output_view, long int a) {
```

Comment:
A separate coding style issue (which I raised previously): new scopes (such as function scopes) should be opened on a new line. Read: please put the opening bracket onto its own line.